client/overview/modal/reset-account/strings.tsx

```diff
@@ -4,12 +4,22 @@
  */
 import { __, sprintf } from '@wordpress/i18n';
 
+/**
+ * Internal dependencies
+ */
+import { isInDevMode } from 'utils';
+
 export default {
 	title: __( 'Reset account', 'woocommerce-payments' ),
-	description: __(
-		'If you are experiencing problems completing account setup, or need to change the email/country associated with your account, you can reset your account and start from the beginning.',
-		'woocommerce-payments'
-	),
+	description: isInDevMode()
+		? __(
+				'If you want to start your testing over, you can reset your account and onboard again.',
```

Comment:
I would change the copy here to:
"In sandbox mode, you can reset your account and onboard again at any time. Please note that all current WooPayments account details, test transactions, and deposits history will be lost."

Reply:
Thanks! Fixed as part of https://github.com/Automattic/woocommerce-payments/pull/8425/commits/8672f742f5b0b328cea9e7c73647ca6f5edd3711